ldk/javascript/src/webpack/generate-banner.test.ts

```diff
@@ -45,9 +75,39 @@ describe('Generate Banner', () => {
     const actual = getLoopMetadataContent(generateBanner(ldkSettings));
     const expected = {
       oliveHelpsContractVersion: '0.3.0',
+      configSchema: {
+        $schema: 'http://json-schema.org/draft-07/schema#',
```

Comment:
This is fine that is here but we shouldn't require this property, I assume leaving this off still works fine correct?

Reply:
Yep! Everything will go through fine without that, it's not required